core/src/test/scala/no/nrk/bigquery/BQValueHasherTest.scala

```diff
@@ -0,0 +1,16 @@
+package no.nrk.bigquery
+
+import munit.FunSuite
+
+import java.util.UUID
+
+class BQValueHasherTest extends FunSuite {
```

Comment:
we should probably also have a BQ Smoketest which checks that the field hashed and our locally hashed version is the same.